drivers/sensors/pmw3360.c

```diff
@@ -86,23 +89,18 @@ void print_byte(uint8_t byte) { dprintf("%c%c%c%c%c%c%c%c|", (byte & 0x80 ? '1'
 #endif
 #define constrain(amt, low, high) ((amt) < (low) ? (low) : ((amt) > (high) ? (high) : (amt)))
 
-bool spi_start_adv(void) {
-    bool status = spi_start(PMW3360_CS_PIN, PMW3360_SPI_LSBFIRST, PMW3360_SPI_MODE, PMW3360_SPI_DIVISOR);
+void pmw3360_spi_start(void) {
```

Comment:
Please revert this to a bool, and keep the return status.  There are some improvements/better handling I have planned that does depend on this.

Reply:
Could you outline roughly what you have planned? Should we change all `pointing_device_driver_t` to return a bool?

So I'm trying to add multi-device support, and was thinking of having the user set:
```
#define PMW3360_CS_PINS { B5, B6 }
```

and then just looping over them for init and firmware upload, and merging their reports either in `quantum/pointing_device_drivers.c` or in the driver itself. Does that sound like a useful approach?

Reply:
I think discussion for adding multiple sensors should be added later. 

But the plans I have are to basically block queries if there are errors.  That way, it isn't constantly calling the sensor code if it isn't connected or is disconnected. 

but pin vs pins is probably the best way to go about it. But it will require a significant rewrite to the code, I think.